CMakeLists.txt

```diff
@@ -29,6 +29,13 @@ cmake_minimum_required(VERSION 2.6)
 project(gnuradio CXX C)
 enable_testing()
 
+# Apple only - look for libraries installed by macports
+if(APPLE)
```

Comment:
There is an "if(APPLE)" a bit further down. Could this instead go there? Seems like keeping those together would beneficial.